tests/e2e/specs/modules/analytics/setup-no-account-with-tag.test.js

```diff
@@ -21,22 +21,25 @@ async function proceedToSetUpAnalytics() {
 	await Promise.all( [
 		expect( page ).toClick( '.googlesitekit-cta-link', { text: /set up analytics/i } ),
 		page.waitForSelector( '.googlesitekit-setup-module--analytics' ),
-		page.waitForResponse( ( res ) => res.url().match( 'analytics/data' ) ),
+		page.waitForResponse( ( res ) => res.url().match( 'analytics/data/accounts-properties-profiles' ) ),
 	] );
 }
 
-const EXISTING_PROPERTY_ID = 'UA-00000001-1';
+const existingTag = {
+	accountID: '999',
+	propertyID: 'UA-999-9',
+};
 
 describe( 'setting up the Analytics module with no existing account and with an existing tag', () => {
 	beforeAll( async () => {
 		await page.setRequestInterception( true );
 		useRequestInterception( ( request ) => {
 			if ( request.url().match( 'modules/analytics/data/tag-permission' ) ) {
 				request.respond( {
-					status: 403,
+					status: 200,
```

Comment:
I could've sworn I got a `403` myself when I got that response locally. Is `200` what the API would send back for a `permission: false` tag?

Reply:
This was changed in #1317, so all good here. The idea is that the sole purpose of this endpoint is checking whether the user has permission to the tag, so returning a 403 indicating a permission error for checking the permission would be kind of meta :)